apps/web/components/ui/ProductCardVertical/ProductCardVertical.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div
+    class="flex border-t-[1px] border-neutral-200 md:border-0 min-w-[320px] max-w-[470px] pt-4"
```

Comment:
```suggestion
    class="flex border-t border-neutral-200 md:border-0 min-w-[320px] max-w-[470px] pt-4"
```
border-t default value is 1px 
![image](https://github.com/vuestorefront/storefront-nuxt3-boilerplate/assets/2566152/fa4b4b10-0d8f-4f42-b580-76ff25160a3c)

Reply:
https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/117/commits/ea5c8af1f118846ff2148db9221626fb2151aaf3